AWSSDKSwiftCLI/Sources/AWSSDKSwiftCLI/Resources/Package.Base.txt

```diff
@@ -56,15 +43,50 @@ let package = Package(
         .tvOS(.v13),
         .watchOS(.v6)
     ],
-    products: [
-        .library(name: "AWSClientRuntime", targets: ["AWSClientRuntime"]),
-        .library(name: "AWSSDKCommon", targets: ["AWSSDKCommon"]),
-        .library(name: "AWSSDKEventStreamsAuth", targets: ["AWSSDKEventStreamsAuth"]),
-        .library(name: "AWSSDKHTTPAuth", targets: ["AWSSDKHTTPAuth"]),
-        .library(name: "AWSSDKIdentity", targets: ["AWSSDKIdentity"]),
-        .library(name: "AWSSDKChecksums", targets: ["AWSSDKChecksums"]),
-    ],
-    targets: [
+    products:
+        runtimeProducts +
+        serviceTargets.map(productForService(_:)),
+    dependencies:
+        [clientRuntimeDependency, crtDependency, doccDependencyOrNil].compactMap { $0 },
+    targets:
+        runtimeTargets +
+        runtimeTestTargets +
+        serviceTargets.map(target(_:)) +
+        serviceTargets.map(unitTestTarget(_:))
+)
```

Comment:
The `Package` instance for this manifest is above.  Note that its params are either a literal or some combination of function or read-only var results.